packages/ui/src/Dialog/index.tsx

```diff
@@ -1,59 +1,43 @@
 import { createContext, ReactNode, useContext } from 'react';
-import styled, { keyframes } from 'styled-components';
+import styled from 'styled-components';
 import * as RadixDialog from '@radix-ui/react-dialog';
 import { Text } from '../Text';
 import { Icon } from '../Icon';
 import { X } from 'lucide-react';
 import { ButtonGroup, ButtonGroupProps } from '../ButtonGroup';
-
-const gradualBlur = (blur: string) => keyframes`
```

Comment:
This animation was causing problems with the `backdrop-filter` of the dialog background, so I nixed it for now. Apparently this is a known limitation of `backdrop-filter` in Chrome.